Monitoring breaks w/ Libraries-BOM 9.0.0

Found in #3534 
```
------------------------------------------------------------
- testing monitoring/cloud-client
------------------------------------------------------------
[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0.509 s <<< FAILURE! - in com.example.monitoring.QuickstartSampleIT
[ERROR] com.example.monitoring.QuickstartSampleIT.testQuickstart  Time elapsed: 0.314 s  <<< ERROR!
java.lang.NoClassDefFoundError: com/google/auth/oauth2/ServiceAccountJwtAccessCredentials
	at com.example.monitoring.QuickstartSampleIT.testQuickstart(QuickstartSampleIT.java:63)
Caused by: java.lang.ClassNotFoundException: com.google.auth.oauth2.ServiceAccountJwtAccessCredentials
	at com.example.monitoring.QuickstartSampleIT.testQuickstart(QuickstartSampleIT.java:63)

[ERROR] Tests run: 8, Failures: 0, Errors: 8, Skipped: 0, Time elapsed: 0.119 s <<< FAILURE! - in com.example.monitoring.SnippetsIT
[ERROR] com.example.monitoring.SnippetsIT.testGetMetricsDescriptor  Time elapsed: 0.075 s  <<< ERROR!
java.lang.NoClassDefFoundError: com/google/auth/oauth2/ServiceAccountJwtAccessCredentials
	at com.example.monitoring.SnippetsIT.testGetMetricsDescriptor(SnippetsIT.java:77)
Caused by: java.lang.ClassNotFoundException: com.google.auth.oauth2.ServiceAccountJwtAccessCredentials
	at com.example.monitoring.SnippetsIT.testGetMetricsDescriptor(SnippetsIT.java:77)

[ERROR] com.example.monitoring.SnippetsIT.testListTimeSeriesHeader  Time elapsed: 0.004 s  <<< ERROR!
java.lang.NoClassDefFoundError: com/google/auth/oauth2/ServiceAccountJwtAccessCredentials
	at com.example.monitoring.SnippetsIT.testListTimeSeriesHeader(SnippetsIT.java:103)
Caused by: java.lang.ClassNotFoundException: com.google.auth.oauth2.ServiceAccountJwtAccessCredentials
	at com.example.monitoring.SnippetsIT.testListTimeSeriesHeader(SnippetsIT.java:103)

[ERROR] com.example.monitoring.SnippetsIT.testListResources  Time elapsed: 0.003 s  <<< ERROR!
java.lang.NoClassDefFoundError: com/google/auth/oauth2/ServiceAccountJwtAccessCredentials
	at com.example.monitoring.SnippetsIT.testListResources(SnippetsIT.java:155)
Caused by: java.lang.ClassNotFoundException: com.google.auth.oauth2.ServiceAccountJwtAccessCredentials
	at com.example.monitoring.SnippetsIT.testListResources(SnippetsIT.java:155)

[ERROR] com.example.monitoring.SnippetsIT.testListTimeSeriesReduce  Time elapsed: 0.003 s  <<< ERROR!
java.lang.NoClassDefFoundError: com/google/auth/oauth2/ServiceAccountJwtAccessCredentials
	at com.example.monitoring.SnippetsIT.testListTimeSeriesReduce(SnippetsIT.java:129)
Caused by: java.lang.ClassNotFoundException: com.google.auth.oauth2.ServiceAccountJwtAccessCredentials
	at com.example.monitoring.SnippetsIT.testListTimeSeriesReduce(SnippetsIT.java:129)

[ERROR] com.example.monitoring.SnippetsIT.testListTimeSeriesAggregate  Time elapsed: 0.003 s  <<< ERROR!
java.lang.NoClassDefFoundError: com/google/auth/oauth2/ServiceAccountJwtAccessCredentials
	at com.example.monitoring.SnippetsIT.testListTimeSeriesAggregate(SnippetsIT.java:116)
Caused by: java.lang.ClassNotFoundException: com.google.auth.oauth2.ServiceAccountJwtAccessCredentials
	at com.example.monitoring.SnippetsIT.testListTimeSeriesAggregate(SnippetsIT.java:116)

[ERROR] com.example.monitoring.SnippetsIT.testGetResource  Time elapsed: 0.003 s  <<< ERROR!
java.lang.NoClassDefFoundError: com/google/auth/oauth2/ServiceAccountJwtAccessCredentials
	at com.example.monitoring.SnippetsIT.testGetResource(SnippetsIT.java:142)
Caused by: java.lang.ClassNotFoundException: com.google.auth.oauth2.ServiceAccountJwtAccessCredentials
	at com.example.monitoring.SnippetsIT.testGetResource(SnippetsIT.java:142)

[ERROR] com.example.monitoring.SnippetsIT.testListMetricsDescriptor  Time elapsed: 0.003 s  <<< ERROR!
java.lang.NoClassDefFoundError: com/google/auth/oauth2/ServiceAccountJwtAccessCredentials
	at com.example.monitoring.SnippetsIT.testListMetricsDescriptor(SnippetsIT.java:65)
Caused by: java.lang.ClassNotFoundException: com.google.auth.oauth2.ServiceAccountJwtAccessCredentials
	at com.example.monitoring.SnippetsIT.testListMetricsDescriptor(SnippetsIT.java:65)

[ERROR] com.example.monitoring.SnippetsIT.testListTimeSeries  Time elapsed: 0.003 s  <<< ERROR!
java.lang.NoClassDefFoundError: com/google/auth/oauth2/ServiceAccountJwtAccessCredentials
	at com.example.monitoring.SnippetsIT.testListTimeSeries(SnippetsIT.java:90)
Caused by: java.lang.ClassNotFoundException: com.google.auth.oauth2.ServiceAccountJwtAccessCredentials
	at com.example.monitoring.SnippetsIT.testListTimeSeries(SnippetsIT.java:90)

[ERROR] Errors:
[ERROR]   QuickstartSampleIT.testQuickstart:63 » NoClassDefFound com/google/auth/oauth2/...
[ERROR]   SnippetsIT.testGetMetricsDescriptor:77 » NoClassDefFound com/google/auth/oauth...
[ERROR]   SnippetsIT.testGetResource:142 » NoClassDefFound com/google/auth/oauth2/Servic...
[ERROR]   SnippetsIT.testListMetricsDescriptor:65 » NoClassDefFound com/google/auth/oaut...
[ERROR]   SnippetsIT.testListResources:155 » NoClassDefFound com/google/auth/oauth2/Serv...
[ERROR]   SnippetsIT.testListTimeSeries:90 » NoClassDefFound com/google/auth/oauth2/Serv...
[ERROR]   SnippetsIT.testListTimeSeriesAggregate:116 » NoClassDefFound com/google/auth/o...
[ERROR]   SnippetsIT.testListTimeSeriesHeader:103 » NoClassDefFound com/google/auth/oaut...
[ERROR]   SnippetsIT.testListTimeSeriesReduce:129 » NoClassDefFound com/google/auth/oaut...
[ERROR] Tests run: 9, Failures: 0, Errors: 9, Skipped: 0
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-failsafe-plugin:3.0.0-M4:verify (default) on project monitoring-google-cloud-samples: There are test failures.

------------------------------------------------------------
- testing monitoring/v3
------------------------------------------------------------
[ERROR] Tests run: 6, Failures: 0, Errors: 6, Skipped: 0, Time elapsed: 0.732 s <<< FAILURE! - in com.example.UptimeIT
[ERROR] com.example.UptimeIT.test1_CreateUptimeCheck  Time elapsed: 0.484 s  <<< ERROR!
java.lang.NoClassDefFoundError: com/google/auth/oauth2/ServiceAccountJwtAccessCredentials
	at com.example.UptimeIT.test1_CreateUptimeCheck(UptimeIT.java:62)
Caused by: java.lang.ClassNotFoundException: com.google.auth.oauth2.ServiceAccountJwtAccessCredentials
	at com.example.UptimeIT.test1_CreateUptimeCheck(UptimeIT.java:62)

[ERROR] com.example.UptimeIT.test2_GetUptimeCheck  Time elapsed: 0.001 s  <<< ERROR!
java.lang.NoClassDefFoundError: com/google/auth/oauth2/ServiceAccountJwtAccessCredentials
	at com.example.UptimeIT.test2_GetUptimeCheck(UptimeIT.java:74)
Caused by: java.lang.ClassNotFoundException: com.google.auth.oauth2.ServiceAccountJwtAccessCredentials
	at com.example.UptimeIT.test2_GetUptimeCheck(UptimeIT.java:74)

[ERROR] com.example.UptimeIT.test2_ListUptimeChecks  Time elapsed: 0.006 s  <<< ERROR!
java.lang.NoClassDefFoundError: com/google/auth/oauth2/ServiceAccountJwtAccessCredentials
	at com.example.UptimeIT.test2_ListUptimeChecks(UptimeIT.java:80)
Caused by: java.lang.ClassNotFoundException: com.google.auth.oauth2.ServiceAccountJwtAccessCredentials
	at com.example.UptimeIT.test2_ListUptimeChecks(UptimeIT.java:80)

[ERROR] com.example.UptimeIT.test2_ListUptimeIps  Time elapsed: 0.003 s  <<< ERROR!
java.lang.NoClassDefFoundError: com/google/auth/oauth2/ServiceAccountJwtAccessCredentials
	at com.example.UptimeIT.test2_ListUptimeIps(UptimeIT.java:87)
Caused by: java.lang.ClassNotFoundException: com.google.auth.oauth2.ServiceAccountJwtAccessCredentials
	at com.example.UptimeIT.test2_ListUptimeIps(UptimeIT.java:87)

[ERROR] com.example.UptimeIT.test2_UpdateUptimeCheck  Time elapsed: 0.027 s  <<< ERROR!
java.lang.NoClassDefFoundError: com/google/auth/oauth2/ServiceAccountJwtAccessCredentials
	at com.example.UptimeIT.test2_UpdateUptimeCheck(UptimeIT.java:68)
Caused by: java.lang.ClassNotFoundException: com.google.auth.oauth2.ServiceAccountJwtAccessCredentials
	at com.example.UptimeIT.test2_UpdateUptimeCheck(UptimeIT.java:68)

[ERROR] com.example.UptimeIT.test3_DeleteUptimeCheck  Time elapsed: 0.002 s  <<< ERROR!
java.lang.NoClassDefFoundError: com/google/auth/oauth2/ServiceAccountJwtAccessCredentials
	at com.example.UptimeIT.test3_DeleteUptimeCheck(UptimeIT.java:97)
Caused by: java.lang.ClassNotFoundException: com.google.auth.oauth2.ServiceAccountJwtAccessCredentials
	at com.example.UptimeIT.test3_DeleteUptimeCheck(UptimeIT.java:97)

[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0.003 s <<< FAILURE! - in com.example.DeleteNotificationChannelIT
[ERROR] com.example.DeleteNotificationChannelIT  Time elapsed: 0 s  <<< ERROR!
java.lang.NoClassDefFoundError: com/google/auth/oauth2/ServiceAccountJwtAccessCredentials
	at com.example.DeleteNotificationChannelIT.setupClass(DeleteNotificationChannelIT.java:56)
Caused by: java.lang.ClassNotFoundException: com.google.auth.oauth2.ServiceAccountJwtAccessCredentials
	at com.example.DeleteNotificationChannelIT.setupClass(DeleteNotificationChannelIT.java:56)

[ERROR] Tests run: 5, Failures: 0, Errors: 4, Skipped: 1, Time elapsed: 0.115 s <<< FAILURE! - in com.example.AlertIT
[ERROR] com.example.AlertIT.testListPolicies  Time elapsed: 0.104 s  <<< ERROR!
java.lang.NoClassDefFoundError: com/google/auth/oauth2/ServiceAccountJwtAccessCredentials
	at com.example.AlertIT.testListPolicies(AlertIT.java:64)
Caused by: java.lang.ClassNotFoundException: com.google.auth.oauth2.ServiceAccountJwtAccessCredentials
	at com.example.AlertIT.testListPolicies(AlertIT.java:64)

[ERROR] com.example.AlertIT.testDisableEnablePolicies  Time elapsed: 0.001 s  <<< ERROR!
java.lang.NoClassDefFoundError: com/google/auth/oauth2/ServiceAccountJwtAccessCredentials
	at com.example.AlertIT.testDisableEnablePolicies(AlertIT.java:98)
Caused by: java.lang.ClassNotFoundException: com.google.auth.oauth2.ServiceAccountJwtAccessCredentials
	at com.example.AlertIT.testDisableEnablePolicies(AlertIT.java:98)

[ERROR] com.example.AlertIT.testBackupPolicies  Time elapsed: 0.001 s  <<< ERROR!
java.lang.NoClassDefFoundError: com/google/auth/oauth2/ServiceAccountJwtAccessCredentials
	at com.example.AlertIT.testBackupPolicies(AlertIT.java:70)
Caused by: java.lang.ClassNotFoundException: com.google.auth.oauth2.ServiceAccountJwtAccessCredentials
	at com.example.AlertIT.testBackupPolicies(AlertIT.java:70)

[ERROR] com.example.AlertIT.testReplaceChannels  Time elapsed: 0.002 s  <<< ERROR!
java.lang.NoClassDefFoundError: com/google/auth/oauth2/ServiceAccountJwtAccessCredentials
	at com.example.AlertIT.testReplaceChannels(AlertIT.java:85)
Caused by: java.lang.ClassNotFoundException: com.google.auth.oauth2.ServiceAccountJwtAccessCredentials
	at com.example.AlertIT.testReplaceChannels(AlertIT.java:85)

[ERROR] Errors:
[ERROR]   AlertIT.testBackupPolicies:70 » NoClassDefFound com/google/auth/oauth2/Service...
[ERROR]   AlertIT.testDisableEnablePolicies:98 » NoClassDefFound com/google/auth/oauth2/...
[ERROR]   AlertIT.testListPolicies:64 » NoClassDefFound com/google/auth/oauth2/ServiceAc...
[ERROR]   AlertIT.testReplaceChannels:85 » NoClassDefFound com/google/auth/oauth2/Servic...
[ERROR]   DeleteNotificationChannelIT.setupClass:56 » NoClassDefFound com/google/auth/oa...
[ERROR]   UptimeIT.test1_CreateUptimeCheck:62 » NoClassDefFound com/google/auth/oauth2/S...
[ERROR]   UptimeIT.test2_GetUptimeCheck:74 » NoClassDefFound com/google/auth/oauth2/Serv...
[ERROR]   UptimeIT.test2_ListUptimeChecks:80 » NoClassDefFound com/google/auth/oauth2/Se...
[ERROR]   UptimeIT.test2_ListUptimeIps:87 » NoClassDefFound com/google/auth/oauth2/Servi...
[ERROR]   UptimeIT.test2_UpdateUptimeCheck:68 » NoClassDefFound com/google/auth/oauth2/S...
[ERROR]   UptimeIT.test3_DeleteUptimeCheck:97 » NoClassDefFound com/google/auth/oauth2/S...
[ERROR] Tests run: 12, Failures: 0, Errors: 11, Skipped: 1
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-failsafe-plugin:3.0.0-M4:verify (default) on project monitoring-google-cloud-v3-samples: There are test failures.
```

--- a/monitoring/cloud-client/pom.xml
+++ b/monitoring/cloud-client/pom.xml
@@ -39,7 +39,7 @@
       <dependency>
         <groupId>com.google.cloud</groupId>
         <artifactId>libraries-bom</artifactId>
-        <version>8.1.0</version>
+        <version>9.0.0</version>
         <type>pom</type>
         <scope>import</scope>
       </dependency>
@@ -66,7 +66,18 @@
       <artifactId>protobuf-java-util</artifactId>
       <version>4.0.0-rc-2</version>
     </dependency>
-  
+
+    <!-- FIXME: remove after client fixes depenency issue => BOM 9.0.0   -->
+    <dependency>
+      <groupId>com.google.auth</groupId>
+      <artifactId>google-auth-library-credentials</artifactId>
+      <version>0.21.1</version>
+    </dependency>
+    <dependency>
+      <groupId>com.google.auth</groupId>
+      <artifactId>google-auth-library-oauth2-http</artifactId>
+      <version>0.21.1</version>
+    </dependency>
 
     <!-- Test dependencies -->
     <dependency>

--- a/monitoring/v3/pom.xml
+++ b/monitoring/v3/pom.xml
@@ -40,7 +40,7 @@
         <dependency>
           <groupId>com.google.cloud</groupId>
           <artifactId>libraries-bom</artifactId>
-          <version>8.1.0</version>
+          <version>9.0.0</version>
           <type>pom</type>
           <scope>import</scope>
         </dependency>
@@ -61,6 +61,18 @@
             <groupId>com.google.code.gson</groupId>
             <artifactId>gson</artifactId>
             <version>2.8.6</version>
+        </dependency>
+
+        <!-- FIXME: remove after client fixes depenency issue => BOM 9.0.0   -->
+        <dependency>
+            <groupId>com.google.auth</groupId>
+            <artifactId>google-auth-library-credentials</artifactId>
+            <version>0.21.1</version>
+        </dependency>
+        <dependency>
+            <groupId>com.google.auth</groupId>
+            <artifactId>google-auth-library-oauth2-http</artifactId>
+            <version>0.21.1</version>
         </dependency>
 
         <!-- Test dependencies -->

